test/Microsoft.AspNet.Mvc.Core.Test/Filters/SkipStatusCodePagesAttributeTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Routing;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core.Test
+{
+    public class SkipStatusCodePagesAttributeTest
+    {
+        [Fact]
+        public async Task SkipStatusCodePagesAttribute_TurnsOfStatusCodePages()
+        {
+            // Arrange
+            var skipStatusCodeAttribute = new SkipStatusCodePagesAttribute();
+            var resultExecutingContext = CreateResultExecutingContext(new IFilter[] { skipStatusCodeAttribute });
+            var statusCodePagesFeature = new TestStatusCodeFeature();
+            resultExecutingContext.HttpContext.SetFeature<IStatusCodePagesFeature>(statusCodePagesFeature);
+            var next = new ResultExecutionDelegate(
+                            () => Task.FromResult(CreateResultExecutedContext(resultExecutingContext)));
+
+            // Act
+            await skipStatusCodeAttribute.OnResultExecutionAsync(resultExecutingContext, next);
+
+            // Assert
+            Assert.Equal(false, statusCodePagesFeature.Enabled);
+        }
+
+        private static ResultExecutedContext CreateResultExecutedContext(ResultExecutingContext context)
```

Comment:
Need another test to verify it no-ops \ doesn't null ref if the feature is missing.